cfgov/cfgov/settings/base.py

```diff
@@ -134,7 +135,6 @@
 
 # Sheer related settings
 
-SHEER_SITES = [Path(os.path.dirname(__file__)).ancestor(2) + '/v1/jinja2/v1',
-               Path(__file__).ancestor(4).child('docs'), STATIC_ROOT]
+SHEER_SITES = [V1_TEMPLATE_ROOT, Path(__file__).ancestor(4).child('docs'), STATIC_ROOT]
```

Comment:
One part we do occasionally reference is `/docs/sheer-layouts/`, but yeah overall we don't use it. At some point we'd like to bring it back though, if that matters for this code.